Tests/SentryTests/Integrations/Performance/IO/SentryNSDataTrackerTests.swift

```diff
@@ -220,12 +263,14 @@ class SentryNSDataTrackerTests: XCTestCase {
         return result?.first
     }
     
-    private func assertDataSpan(_ span: Span?, path: String, operation: String, size: Int ) {
+    private func assertDataSpan(_ span: Span?, path: String, operation: String, size: Int, mainThread: Bool = true ) {
         XCTAssertNotNil(span)
         XCTAssertEqual(span?.operation, operation)
         XCTAssertTrue(span?.isFinished ?? false)
         XCTAssertEqual(span?.data["file.size"] as? Int, size)
         XCTAssertEqual(span?.data["file.path"] as? String, path)
+        XCTAssertEqual(span?.data["blocked_main_thread"] as? Bool ?? false, mainThread)
+        XCTAssertEqual(span?.data["call_stack"] != nil, mainThread)
```

Comment:
`m`: This assertion doesn't validate much. I think it would be better to use a `TestSentryThreadInspector` and assert that `SentryNSDataTracker` properly assigns the stacktrace of the `TestSentryThreadInspector`. We could also validate then that it skips system frames.